flycheck-xo.el

```diff
@@ -39,33 +39,50 @@
 
 (flycheck-def-args-var flycheck-xo-args xo)
 
-(flycheck-define-checker xo
-  "A checker using xo.
+(defun flycheck-parse-xo (output checker buffer)
```

Comment:
The more conventional name would be flycheck-xo-parse since is then using the package namespace prefix - can you please change it?

Reply:
done and done